
<!DOCTYPE html>


<html lang="en" data-content_root="./" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Markdown Files &#8212; Smart Microscopy Working Group&#39;s Website</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="_static/styles/theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="_static/styles/bootstrap.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="_static/styles/pydata-sphinx-theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />

  
  <link href="_static/vendor/fontawesome/6.5.2/css/all.min.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.5.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.5.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.5.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=03e43079" />
    <link rel="stylesheet" type="text/css" href="_static/styles/sphinx-book-theme.css?v=eba8b062" />
    <link rel="stylesheet" type="text/css" href="_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="_static/mystnb.8ecb98da25f57f5357bf6f572d296f466b2cfe2517ffebfabe82451661e28f02.css" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-thebe.css?v=4fa983c6" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-design.min.css?v=95c83b7e" />
    <link rel="stylesheet" type="text/css" href="_static/custom.css?v=0915d2f8" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b" />
<link rel="preload" as="script" href="_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b" />
  <script src="_static/vendor/fontawesome/6.5.2/js/all.min.js?digest=dfe6caa3a7d634c4db9b"></script>

    <script src="_static/documentation_options.js?v=9eb32ce0"></script>
    <script src="_static/doctools.js?v=9a2dae69"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="_static/copybutton.js?v=f281be69"></script>
    <script src="_static/scripts/sphinx-book-theme.js?v=887ef09a"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="_static/togglebutton.js?v=4a39c7ea"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="_static/design-tabs.js?v=f930bc37"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script async="async" src="_static/sphinx-thebe.js?v=c100c467"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'markdown';</script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-primary-sidebar-checkbox"/>
  <label class="overlay overlay-primary" for="pst-primary-sidebar-checkbox"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-secondary-sidebar-checkbox"/>
  <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
        
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">

  
    
  

<a class="navbar-brand logo" href="index.html">
  
  
  
  
  
    
    
      
    
    
    <img src="_static/logo.png" class="logo__image only-light" alt="Smart Microscopy Working Group's Website - Home"/>
    <script>document.write(`<img src="_static/logo.png" class="logo__image only-dark" alt="Smart Microscopy Working Group's Website - Home"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item">

 <script>
 document.write(`
   <button class="btn search-button-field search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
    <span class="search-button__default-text">Search</span>
    <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
   </button>
 `);
 </script></div>
        <div class="sidebar-primary-item"><nav class="bd-links bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="index.html">
                    Smart Microscopy Working Group
                </a>
            </li>
        </ul>
        <ul class="nav bd-sidenav">
<li class="toctree-l1 has-children"><a class="reference internal" href="members.html">Members of the Group</a><details><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/CamachoR.html">Rafael Camacho</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/PfanderC.html">Claudia Pfander</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/ThomasL.html">Laurent Thomas</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/MartinC.html">Célia Martin</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/BoucharebO.html">Otmane Bouchareb</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/DemeautisC.html">Claire Demeautis</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/GuilbertT.html">Thomas Guilbert</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/TorroR.html">Rémy Torro</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/NorlinN.html">Nils Norlin</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/HeilH.html">Hannah S. Heil</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/HinderlingL.html">Lucien Hinderling</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/HalavatyiA.html">Aliaksandr Halavatyi</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/DiederichB.html">Benedict Diederich</a></li>
<li class="toctree-l2"><a class="reference internal" href="members/profiles/BrooksS.html">Scott Brooks</a></li>
</ul>
</details></li>
<li class="toctree-l1 has-children"><a class="reference internal" href="implementations.html">Available implementations</a><details><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul>
<li class="toctree-l2"><a class="reference internal" href="implementations/academic/unibe.html">rtm-pymmcore: Automation of optogenetic targeting</a></li>
<li class="toctree-l2"><a class="reference internal" href="implementations/academic/cci-ug.html">Smart Microscopy as a Service</a></li>
<li class="toctree-l2"><a class="reference internal" href="implementations/academic/uu.html">Outcome-driven microscopy to guide cell biology</a></li>
<li class="toctree-l2"><a class="reference internal" href="implementations/academic/uow.html">CelFDrive: Multimodal AI-Assisted Microscopy for Detection of Rare Events</a></li>
<li class="toctree-l2"><a class="reference internal" href="implementations/academic/embl.html">Modular adaptive feedback microscopy workflows with AutoMicTools</a></li>
<li class="toctree-l2"><a class="reference internal" href="implementations/academic/cnrs.html">The EnderScope as an educational platform for Smart Microscopy</a></li>
<li class="toctree-l2"><a class="reference internal" href="implementations/academic/princeton.html">PyCLM: programming-free, closed-loop microscopy for optogenetic stimulation</a></li>
<li class="toctree-l2"><a class="reference internal" href="implementations/industry/inscoper.html">Inscoper</a></li>
<li class="toctree-l2"><a class="reference internal" href="implementations/industry/zeiss.html">Open Application Development (OAD) in Zeiss ZEN</a></li>
<li class="toctree-l2"><a class="reference internal" href="implementations/industry/openUC2.html">ImSwitch for openUC2</a></li>





<li class="toctree-l2"><a class="reference internal" href="implementations/industry/cytely.html">Automation of spatial single-cell analysis with Cytely</a></li>
<li class="toctree-l2"><a class="reference internal" href="implementations/industry/nikon.html">Visual programming for Smart Microscopy with NIS-Elements (Nikon)</a></li>
</ul>
</details></li>
<li class="toctree-l1"><a class="reference internal" href="notes.html">Public Notes</a></li>
<li class="toctree-l1"><a class="reference internal" href="publications.html">Publications &amp; Outreach</a></li>
<li class="toctree-l1 has-children"><a class="reference internal" href="trainingmaterials/index.html">Training Materials (in development)</a><details><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul>
<li class="toctree-l2"><a class="reference internal" href="trainingmaterials/guidelines.html">Guidelines for contribution</a></li>
<li class="toctree-l2"><a class="reference internal" href="trainingmaterials/modules.html">Smart microscopy training modules</a></li>
</ul>
</details></li>
</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><button class="sidebar-toggle primary-toggle btn btn-sm" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</button></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">


<a href="https://github.com/SmartMicroscopy/smartmicroscopy.github.io" target="_blank"
   class="btn btn-sm btn-source-repository-button"
   title="Source repository"
   data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>

</a>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="_sources/markdown.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<script>
document.write(`
  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
  </button>
`);
</script>


<script>
document.write(`
  <button class="btn btn-sm pst-navbar-icon search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass fa-lg"></i>
  </button>
`);
</script>
<button class="sidebar-toggle secondary-toggle btn btn-sm" title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</button>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Markdown Files</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#what-is-myst">What is MyST?</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sample-roles-and-directives">Sample Roles and Directives</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#citations">Citations</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#learn-more">Learn more</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article">
                  
  <section class="tex2jax_ignore mathjax_ignore" id="markdown-files">
<h1>Markdown Files<a class="headerlink" href="#markdown-files" title="Link to this heading">#</a></h1>
<p>Whether you write your book’s content in Jupyter Notebooks (<code class="docutils literal notranslate"><span class="pre">.ipynb</span></code>) or
in regular markdown files (<code class="docutils literal notranslate"><span class="pre">.md</span></code>), you’ll write in the same flavor of markdown
called <strong>MyST Markdown</strong>.
This is a simple file to help you get started and show off some syntax.</p>
<section id="what-is-myst">
<h2>What is MyST?<a class="headerlink" href="#what-is-myst" title="Link to this heading">#</a></h2>
<p>MyST stands for “Markedly Structured Text”. It
is a slight variation on a flavor of markdown called “CommonMark” markdown,
with small syntax extensions to allow you to write <strong>roles</strong> and <strong>directives</strong>
in the Sphinx ecosystem.</p>
<p>For more about MyST, see <a class="reference external" href="https://jupyterbook.org/content/myst.html">the MyST Markdown Overview</a>.</p>
</section>
<section id="sample-roles-and-directives">
<h2>Sample Roles and Directives<a class="headerlink" href="#sample-roles-and-directives" title="Link to this heading">#</a></h2>
<p>Roles and directives are two of the most powerful tools in Jupyter Book. They
are kind of like functions, but written in a markup language. They both
serve a similar purpose, but <strong>roles are written in one line</strong>, whereas
<strong>directives span many lines</strong>. They both accept different kinds of inputs,
and what they do with those inputs depends on the specific role or directive
that is being called.</p>
<p>Here is a “note” directive:</p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>Here is a note</p>
</div>
<p>It will be rendered in a special box when you build your book.</p>
<p>Here is an inline directive to refer to a document: <a class="reference internal" href="markdown-notebooks.html"><span class="doc">Notebooks with MyST Markdown</span></a>.</p>
</section>
<section id="citations">
<h2>Citations<a class="headerlink" href="#citations" title="Link to this heading">#</a></h2>
<p>You can also cite references that are stored in a <code class="docutils literal notranslate"><span class="pre">bibtex</span></code> file. For example,
the following syntax: <code class="docutils literal notranslate"><span class="pre">{cite}`Oatman2025`</span></code> will render like
this: <span id="id1">[<a class="reference internal" href="#id41" title="Harrison R. Oatman, Beena C. Lad, and Jared Toettcher. Pyclm: programming-free, closed-loop microscopy for real-time measurement, segmentation, and optogenetic stimulation. bioRxiv, September 2025. URL: http://dx.doi.org/10.1101/2025.08.29.673155, doi:10.1101/2025.08.29.673155.">Oatman <em>et al.</em>, 2025</a>]</span>.</p>
<p>Moreover, you can insert a bibliography into your page with this syntax:
The <code class="docutils literal notranslate"><span class="pre">{bibliography}</span></code> directive must be used for all the <code class="docutils literal notranslate"><span class="pre">{cite}</span></code> roles to
render properly.
For example, if the references for your book are stored in <code class="docutils literal notranslate"><span class="pre">references.bib</span></code>,
then the bibliography is inserted with:</p>
<div class="docutils container" id="id2">
<div role="list" class="citation-list">
<div class="citation" id="id32" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>AKAN+23<span class="fn-bracket">]</span></span>
<p>Oscar André, Johannes Kumra Ahnlide, Nils Norlin, Vinay Swaminathan, and Pontus Nordenfelt. Data-driven microscopy allows for automated context-specific acquisition of high-fidelity image data. <em>Cell Reports Methods</em>, 3(3):100419, March 2023. URL: <a class="reference external" href="http://dx.doi.org/10.1016/j.crmeth.2023.100419">http://dx.doi.org/10.1016/j.crmeth.2023.100419</a>, <a class="reference external" href="https://doi.org/10.1016/j.crmeth.2023.100419">doi:10.1016/j.crmeth.2023.100419</a>.</p>
</div>
<div class="citation" id="id12" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>BSEH+21<span class="fn-bracket">]</span></span>
<p>Maël Balluet, Florian Sizaire, Youssef El Habouz, Thomas Walter, Jérémy Pont, Baptiste Giroux, Otmane Bouchareb, Marc Tramier, and Jacques Pecreaux. Neural network fast‐classifies biological images through features selecting to power automated microscopy. <em>Journal of Microscopy</em>, 285(1):3–19, November 2021. URL: <a class="reference external" href="http://dx.doi.org/10.1111/jmi.13062">http://dx.doi.org/10.1111/jmi.13062</a>, <a class="reference external" href="https://doi.org/10.1111/jmi.13062">doi:10.1111/jmi.13062</a>.</p>
</div>
<div class="citation" id="id5" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>BSD+19<span class="fn-bracket">]</span></span>
<p>Giulia Bertolin, Florian Sizaire, Claire Déméautis, Catherine Chapuis, Fabienne Mérola, Marie Erard, and Marc Tramier. Optimized fret pairs and quantification approaches to detect the activation of aurora kinase a at mitosis. <em>ACS Sensors</em>, 4(8):2018–2027, July 2019. URL: <a class="reference external" href="http://dx.doi.org/10.1021/acssensors.9b00486">http://dx.doi.org/10.1021/acssensors.9b00486</a>, <a class="reference external" href="https://doi.org/10.1021/acssensors.9b00486">doi:10.1021/acssensors.9b00486</a>.</p>
</div>
<div class="citation" id="id10" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>BEHM+24<span class="fn-bracket">]</span></span>
<p>Julia Bonnet, Youssef El-Habouz, Célia Martin, Maelle Guillout, Louis Ruel, Baptiste Giroux, Claire Demeautis, Benjamin Mercat, Otmane Bouchareb, Jacques Pécreaux, and Marc Tramier. The roboscope: smart and fast microscopy for generic event-driven acquisition. <em>bioRxiv</em>, September 2024. URL: <a class="reference external" href="http://dx.doi.org/10.1101/2024.09.24.614735">http://dx.doi.org/10.1101/2024.09.24.614735</a>, <a class="reference external" href="https://doi.org/10.1101/2024.09.24.614735">doi:10.1101/2024.09.24.614735</a>.</p>
</div>
<div class="citation" id="id33" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>BTPC+24<span class="fn-bracket">]</span></span>
<p>Scott Brooks, Sara Toral-Pérez, David S. Corcoran, Karl Kilborn, Brian Bodensteiner, Hella Baumann, Nigel J. Burroughs, Andrew D. McAinsh, and Till Bretschneider. Celfdrive: artificial intelligence assisted microscopy for automated detection of rare events. <em>bioRxiv</em>, October 2024. URL: <a class="reference external" href="http://dx.doi.org/10.1101/2024.10.17.618897">http://dx.doi.org/10.1101/2024.10.17.618897</a>, <a class="reference external" href="https://doi.org/10.1101/2024.10.17.618897">doi:10.1101/2024.10.17.618897</a>.</p>
</div>
<div class="citation" id="id25" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>BMZ+25<span class="fn-bracket">]</span></span>
<p>Andreas Brunner, Natalia Rosalía Morero, Wanlu Zhang, M. Julius Hossain, Marko Lampe, Hannah Pflaumer, Aliaksandr Halavatyi, Jan-Michael Peters, Kai S. Beckwith, and Jan Ellenberg. Quantitative imaging of loop extruders rebuilding interphase genome architecture after mitosis. <em>Journal of Cell Biology</em>, January 2025. URL: <a class="reference external" href="http://dx.doi.org/10.1083/jcb.202405169">http://dx.doi.org/10.1083/jcb.202405169</a>, <a class="reference external" href="https://doi.org/10.1083/jcb.202405169">doi:10.1083/jcb.202405169</a>.</p>
</div>
<div class="citation" id="id26" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>CGEU+24<span class="fn-bracket">]</span></span>
<p>Marco Raffaele Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, Aliaksandr Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, Isidro Cortes-Ciriano, Andreas Kulozik, Rainer Pepperkok, and Jan O. Korbel. Origins ofde novochromosome rearrangements unveiled by coupled imaging and genomics. <em>bioRxiv</em>, August 2024. URL: <a class="reference external" href="http://dx.doi.org/10.1101/2024.08.15.607890">http://dx.doi.org/10.1101/2024.08.15.607890</a>, <a class="reference external" href="https://doi.org/10.1101/2024.08.15.607890">doi:10.1101/2024.08.15.607890</a>.</p>
</div>
<div class="citation" id="id6" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>DSR+17<span class="fn-bracket">]</span></span>
<p>Claire Demeautis, François Sipieter, Julien Roul, Catherine Chapuis, Sergi Padilla-Parra, Franck B. Riquet, and Marc Tramier. Multiplexing pka and erk1&amp;amp;2 kinases fret biosensors in living cells using single excitation wavelength dual colour flim. <em>Scientific Reports</em>, January 2017. URL: <a class="reference external" href="http://dx.doi.org/10.1038/srep41026">http://dx.doi.org/10.1038/srep41026</a>, <a class="reference external" href="https://doi.org/10.1038/srep41026">doi:10.1038/srep41026</a>.</p>
</div>
<div class="citation" id="id36" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>DGU+18<span class="fn-bracket">]</span></span>
<p>Elliot Dine, Agnieszka A. Gil, Giselle Uribe, Clifford P. Brangwynne, and Jared E. Toettcher. Protein phase separation provides long-term memory of transient spatial stimuli. <em>Cell Systems</em>, 6(6):655–663.e5, June 2018. URL: <a class="reference external" href="http://dx.doi.org/10.1016/j.cels.2018.05.002">http://dx.doi.org/10.1016/j.cels.2018.05.002</a>, <a class="reference external" href="https://doi.org/10.1016/j.cels.2018.05.002">doi:10.1016/j.cels.2018.05.002</a>.</p>
</div>
<div class="citation" id="id39" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>EAH+10<span class="fn-bracket">]</span></span>
<p>Arthur Edelstein, Nenad Amodaj, Karl Hoover, Ron Vale, and Nico Stuurman. Computer control of microscopes using µmanager. <em>Current Protocols in Molecular Biology</em>, October 2010. URL: <a class="reference external" href="http://dx.doi.org/10.1002/0471142727.mb1420s92">http://dx.doi.org/10.1002/0471142727.mb1420s92</a>, <a class="reference external" href="https://doi.org/10.1002/0471142727.mb1420s92">doi:10.1002/0471142727.mb1420s92</a>.</p>
</div>
<div class="citation" id="id7" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>FBVD+22<span class="fn-bracket">]</span></span>
<p>Orestis Faklaris, Leslie Bancel-Vallée, Aurélien Dauphin, Baptiste Monterroso, Perrine Frère, David Geny, Tudor Manoliu, Sylvain de Rossi, Fabrice P. Cordelières, Damien Schapman, Roland Nitschke, Julien Cau, and Thomas Guilbert. Quality assessment in light microscopy for routine use through simple tools and robust metrics. <em>Journal of Cell Biology</em>, September 2022. URL: <a class="reference external" href="http://dx.doi.org/10.1083/jcb.202107093">http://dx.doi.org/10.1083/jcb.202107093</a>, <a class="reference external" href="https://doi.org/10.1083/jcb.202107093">doi:10.1083/jcb.202107093</a>.</p>
</div>
<div class="citation" id="id37" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>FYM+23<span class="fn-bracket">]</span></span>
<p>Payam E Farahani, Xiaoyu Yang, Emily V Mesev, Kaylan A Fomby, Ellen H Brumbaugh-Reed, Caleb J Bashor, Celeste M Nelson, and Jared E Toettcher. Pytags enable spatiotemporal measurements of receptor tyrosine kinase signaling in living cells. <em>eLife</em>, May 2023. URL: <a class="reference external" href="http://dx.doi.org/10.7554/eLife.82863">http://dx.doi.org/10.7554/eLife.82863</a>, <a class="reference external" href="https://doi.org/10.7554/elife.82863">doi:10.7554/elife.82863</a>.</p>
</div>
<div class="citation" id="id27" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>FJH+21<span class="fn-bracket">]</span></span>
<p>Timothy Fuqua, Jeff Jordan, Aliaksandr Halavatyi, Christian Tischer, Kerstin Richter, and Justin Crocker. An open-source semi-automated robotics pipeline for embryo immunohistochemistry. <em>Scientific Reports</em>, May 2021. URL: <a class="reference external" href="http://dx.doi.org/10.1038/s41598-021-89676-5">http://dx.doi.org/10.1038/s41598-021-89676-5</a>, <a class="reference external" href="https://doi.org/10.1038/s41598-021-89676-5">doi:10.1038/s41598-021-89676-5</a>.</p>
</div>
<div class="citation" id="id28" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>FJvB+20<span class="fn-bracket">]</span></span>
<p>Timothy Fuqua, Jeff Jordan, Maria Elize van Breugel, Aliaksandr Halavatyi, Christian Tischer, Peter Polidoro, Namiko Abe, Albert Tsai, Richard S. Mann, David L. Stern, and Justin Crocker. Dense and pleiotropic regulatory information in a developmental enhancer. <em>Nature</em>, 587(7833):235–239, October 2020. URL: <a class="reference external" href="http://dx.doi.org/10.1038/s41586-020-2816-5">http://dx.doi.org/10.1038/s41586-020-2816-5</a>, <a class="reference external" href="https://doi.org/10.1038/s41586-020-2816-5">doi:10.1038/s41586-020-2816-5</a>.</p>
</div>
<div class="citation" id="id40" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>GPG+25<span class="fn-bracket">]</span></span>
<p>Sirine Gharbi, Emeric Poiraud, Hugo Le Guenno, Erwan Grandgirard, Charly Rousseau, Niamh Burke, Jerome Mutterer, and David Rousseau. Enderscope.py: a library for computational imaging using the enderscope automated microscope. <em>SoftwareX</em>, 31:102210, September 2025. URL: <a class="reference external" href="http://dx.doi.org/10.1016/j.softx.2025.102210">http://dx.doi.org/10.1016/j.softx.2025.102210</a>, <a class="reference external" href="https://doi.org/10.1016/j.softx.2025.102210">doi:10.1016/j.softx.2025.102210</a>.</p>
</div>
<div class="citation" id="id22" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>HSMB22<span class="fn-bracket">]</span></span>
<p>Jonathan U Harrison, Onur Sen, Andrew D McAinsh, and Nigel J Burroughs. Kinetochore tracking in 3d from lattice light-sheet imaging data with kit. <em>Bioinformatics</em>, 38(12):3315–3317, May 2022. URL: <a class="reference external" href="http://dx.doi.org/10.1093/bioinformatics/btac330">http://dx.doi.org/10.1093/bioinformatics/btac330</a>, <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btac330">doi:10.1093/bioinformatics/btac330</a>.</p>
</div>
<div class="citation" id="id3" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>HHvU+23<span class="fn-bracket">]</span></span>
<p>Max Heydasch, Lucien Hinderling, Jakobus van Unen, Maciej Dobrzynski, and Olivier Pertz. Gtpase activating protein dlc1 spatio-temporally regulates rho signaling. <em>eLife</em>, December 2023. URL: <a class="reference external" href="http://dx.doi.org/10.7554/eLife.90305.1">http://dx.doi.org/10.7554/eLife.90305.1</a>, <a class="reference external" href="https://doi.org/10.7554/elife.90305.1">doi:10.7554/elife.90305.1</a>.</p>
</div>
<div class="citation" id="id18" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>HHK+25<span class="fn-bracket">]</span></span>
<p>Lucien Hinderling, Remo Hadorn, Moritz Kwasny, Joël Frei, Benjamin Grädel, Sacha Psalmon, Yannick Blum, Rémi Berthoz, Alex Landolt, Benjamin D. Towbin, Daniel Riveline, and Olivier Pertz. Teach your microscope how to print: low-cost and rapid-iteration microfabrication for biology. <em>bioRxiv</em>, February 2025. URL: <a class="reference external" href="http://dx.doi.org/10.1101/2025.02.20.639256">http://dx.doi.org/10.1101/2025.02.20.639256</a>, <a class="reference external" href="https://doi.org/10.1101/2025.02.20.639256">doi:10.1101/2025.02.20.639256</a>.</p>
</div>
<div class="citation" id="id20" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>HLGradel+25<span class="fn-bracket">]</span></span>
<p>Lucien Hinderling, Alex E. Landolt, Benjamin Grädel, Laurent Dubied, Cédric Zahni, Moritz Kwasny, Agne Frismantiene, Talley Lambert, Maciej Dobrzyński, and Olivier Pertz. Real-time feedback control microscopy for automation of optogenetic targeting. <em>bioRxiv</em>, August 2025. URL: <a class="reference external" href="http://dx.doi.org/10.1101/2025.08.17.670729">http://dx.doi.org/10.1101/2025.08.17.670729</a>, <a class="reference external" href="https://doi.org/10.1101/2025.08.17.670729">doi:10.1101/2025.08.17.670729</a>.</p>
</div>
<div class="citation" id="id19" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>HWS+24<span class="fn-bracket">]</span></span>
<p>Lucien Hinderling, Guillaume Witz, Roman Schwob, Ana Stojiljkovic, Maciej Dobrzynski, Mykhailo Vladymyrov, Joel Frei, Benjamin Graedel, Agne Frismantiene, and Olivier Pertz. Convpaint - universal framework for interactive pixel classification using pretrained neural networks. <em>bioRxiv</em>, September 2024. URL: <a class="reference external" href="http://dx.doi.org/10.1101/2024.09.12.610926">http://dx.doi.org/10.1101/2024.09.12.610926</a>, <a class="reference external" href="https://doi.org/10.1101/2024.09.12.610926">doi:10.1101/2024.09.12.610926</a>.</p>
</div>
<div class="citation" id="id23" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>JQC23<span class="fn-bracket">]</span></span>
<p>Glenn Jocher, Jing Qiu, and Ayush Chaurasia. Ultralytics YOLO. <em>GitHub</em>, January 2023. URL: <a class="github reference external" href="https://github.com/ultralytics/ultralytics">ultralytics/ultralytics</a>.</p>
</div>
<div class="citation" id="id35" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>KKL+14<span class="fn-bracket">]</span></span>
<p>Nury Kim, Jin Man Kim, Minji Lee, Cha Yeon Kim, Ki-Young Chang, and Won Do Heo. Spatiotemporal control of fibroblast growth factor receptor signals by blue light. <em>Chemistry and Biology</em>, 21(7):903–912, July 2014. URL: <a class="reference external" href="http://dx.doi.org/10.1016/j.chembiol.2014.05.013">http://dx.doi.org/10.1016/j.chembiol.2014.05.013</a>, <a class="reference external" href="https://doi.org/10.1016/j.chembiol.2014.05.013">doi:10.1016/j.chembiol.2014.05.013</a>.</p>
</div>
<div class="citation" id="id14" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>KMR+23<span class="fn-bracket">]</span></span>
<p>Alexander Kirillov, Eric Mintun, Nikhila Ravi, Hanzi Mao, Chloe Rolland, Laura Gustafson, Tete Xiao, Spencer Whitehead, Alexander C. Berg, Wan-Yen Lo, Piotr Dollár, and Ross Girshick. Segment anything. 2023. URL: <a class="reference external" href="https://arxiv.org/abs/2304.02643">https://arxiv.org/abs/2304.02643</a>, <a class="reference external" href="https://doi.org/10.48550/ARXIV.2304.02643">doi:10.48550/ARXIV.2304.02643</a>.</p>
</div>
<div class="citation" id="id29" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>KMR+24<span class="fn-bracket">]</span></span>
<p>Tobias Kletter, Omar Muñoz, Sebastian Reusch, Abin Biswas, Aliaksandr Halavatyi, Beate Neumann, Benno Kuropka, Vasily Zaburdaev, and Simone Reber. Cell state-specific cytoplasmic material properties control spindle architecture and scaling. <em>bioRxiv</em>, July 2024. URL: <a class="reference external" href="http://dx.doi.org/10.1101/2024.07.22.604615">http://dx.doi.org/10.1101/2024.07.22.604615</a>, <a class="reference external" href="https://doi.org/10.1101/2024.07.22.604615">doi:10.1101/2024.07.22.604615</a>.</p>
</div>
<div class="citation" id="id21" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>KGW+24<span class="fn-bracket">]</span></span>
<p>Li Ren Kong, Komal Gupta, Andy Jialun Wu, David Perera, Roland Ivanyi-Nagy, Syed Moiz Ahmed, Tuan Zea Tan, Shawn Lu-Wen Tan, Alessandra Fuddin, Elayanambi Sundaramoorthy, Grace Shiqing Goh, Regina Tong Xin Wong, Ana S.H. Costa, Callum Oddy, Hannan Wong, C. Pawan K. Patro, Yun Suen Kho, Xiao Zi Huang, Joan Choo, Mona Shehata, Soo Chin Lee, Boon Cher Goh, Christian Frezza, Jason J. Pitt, and Ashok R. Venkitaraman. A glycolytic metabolite bypasses “two-hit” tumor suppression by brca2. <em>Cell</em>, 187(9):2269–2287.e16, April 2024. URL: <a class="reference external" href="http://dx.doi.org/10.1016/j.cell.2024.03.006">http://dx.doi.org/10.1016/j.cell.2024.03.006</a>, <a class="reference external" href="https://doi.org/10.1016/j.cell.2024.03.006">doi:10.1016/j.cell.2024.03.006</a>.</p>
</div>
<div class="citation" id="id9" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>MBAH+24<span class="fn-bracket">]</span></span>
<p>David McFadden, Laszlo Barna, Luis-Francisco Acevedo-Hueso, Sergiy V Avilov, Gert-Jan Bakker, Sebastian Beer, Ivan Belyaev, Valeria Berno, Mariana T Carvalho, Yann Cesbron, Catalin Chiritescu, Orestis Faklaris, Nathalie Gaudreault, David Grunwald, Thomas Guilbert, Mathias Hammer, Rainer Heintzmann, Gerhard Holst, Ayse Aslihan Koksoy, Gabriel G Martins, Glyn Nelson, Roland Nitschke, Alex L Payne-Dwyer, Santosh Podder, Sathya Srinivasan, Kees van der Oord, Andre Zeug, and Britta Schroth-Diez. Characterization of the photon conversion factor, noise, and dynamic range of light microscope detection systems v1. <em>protocols.io</em>, September 2024. URL: <a class="reference external" href="http://dx.doi.org/10.17504/protocols.io.14egn61pyl5d/v1">http://dx.doi.org/10.17504/protocols.io.14egn61pyl5d/v1</a>, <a class="reference external" href="https://doi.org/10.17504/protocols.io.14egn61pyl5d/v1">doi:10.17504/protocols.io.14egn61pyl5d/v1</a>.</p>
</div>
<div class="citation" id="id8" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>NBB+21<span class="fn-bracket">]</span></span>
<p>Glyn Nelson, Ulrike Boehm, Steve Bagley, Peter Bajcsy, Johanna Bischof, Claire M. Brown, Aurélien Dauphin, Ian M. Dobbie, John E. Eriksson, Orestis Faklaris, Julia Fernandez‐Rodriguez, Alexia Ferrand, Laurent Gelman, Ali Gheisari, Hella Hartmann, Christian Kukat, Alex Laude, Miso Mitkovski, Sebastian Munck, Alison J. North, Tobias M. Rasse, Ute Resch‐Genger, Lucas C. Schuetz, Arne Seitz, Caterina Strambio‐De‐Castillia, Jason R. Swedlow, Ioannis Alexopoulos, Karin Aumayr, Sergiy Avilov, Gert‐Jan Bakker, Rodrigo R. Bammann, Andrea Bassi, Hannes Beckert, Sebastian Beer, Yury Belyaev, Jakob Bierwagen, Konstantin A. Birngruber, Manel Bosch, Juergen Breitlow, Lisa A. Cameron, Joe Chalfoun, James J. Chambers, Chieh‐Li Chen, Eduardo Conde‐Sousa, Alexander D. Corbett, Fabrice P. Cordelieres, Elaine Del Nery, Ralf Dietzel, Frank Eismann, Elnaz Fazeli, Andreas Felscher, Hans Fried, Nathalie Gaudreault, Wah Ing Goh, Thomas Guilbert, Roland Hadleigh, Peter Hemmerich, Gerhard A. Holst, Michelle S. Itano, Claudia B. Jaffe, Helena K. Jambor, Stuart C. Jarvis, Antje Keppler, David Kirchenbuechler, Marcel Kirchner, Norio Kobayashi, Gabriel Krens, Susanne Kunis, Judith Lacoste, Marco Marcello, Gabriel G. Martins, Daniel J. Metcalf, Claire A. Mitchell, Joshua Moore, Tobias Mueller, Michael S. Nelson, Stephen Ogg, Shuichi Onami, Alexandra L. Palmer, Perrine Paul‐Gilloteaux, Jaime A. Pimentel, Laure Plantard, Santosh Podder, Elton Rexhepaj, Arnaud Royon, Markku A. Saari, Damien Schapman, Vincent Schoonderwoert, Britta Schroth‐Diez, Stanley Schwartz, Michael Shaw, Martin Spitaler, Martin T. Stoeckl, Damir Sudar, Jeremie Teillon, Stefan Terjung, Roland Thuenauer, Christian D. Wilms, Graham D. Wright, and Roland Nitschke. Quarep‐limi: a community‐driven initiative to establish guidelines for quality assessment and reproducibility for instruments and images in light microscopy. <em>Journal of Microscopy</em>, 284(1):56–73, August 2021. URL: <a class="reference external" href="http://dx.doi.org/10.1111/jmi.13041">http://dx.doi.org/10.1111/jmi.13041</a>, <a class="reference external" href="https://doi.org/10.1111/jmi.13041">doi:10.1111/jmi.13041</a>.</p>
</div>
<div class="citation" id="id15" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>NWR+16<span class="fn-bracket">]</span></span>
<p>Dominik Niopek, Pierre Wehler, Julia Roensch, Roland Eils, and Barbara Di Ventura. Optogenetic control of nuclear protein export. <em>Nature Communications</em>, February 2016. URL: <a class="reference external" href="http://dx.doi.org/10.1038/ncomms10624">http://dx.doi.org/10.1038/ncomms10624</a>, <a class="reference external" href="https://doi.org/10.1038/ncomms10624">doi:10.1038/ncomms10624</a>.</p>
</div>
<div class="citation" id="id41" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id1">OLT25</a><span class="fn-bracket">]</span></span>
<p>Harrison R. Oatman, Beena C. Lad, and Jared Toettcher. Pyclm: programming-free, closed-loop microscopy for real-time measurement, segmentation, and optogenetic stimulation. <em>bioRxiv</em>, September 2025. URL: <a class="reference external" href="http://dx.doi.org/10.1101/2025.08.29.673155">http://dx.doi.org/10.1101/2025.08.29.673155</a>, <a class="reference external" href="https://doi.org/10.1101/2025.08.29.673155">doi:10.1101/2025.08.29.673155</a>.</p>
</div>
<div class="citation" id="id17" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>Oga01<span class="fn-bracket">]</span></span>
<p>Katsuhiko Ogata. <em>Modern control engineering</em>. Pearson, Upper Saddle River, NJ, 4 edition, November 2001.</p>
</div>
<div class="citation" id="id13" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>PRSchroder+24<span class="fn-bracket">]</span></span>
<p>Josiah B. Passmore, Alfredo Rates, Jakob Schröder, Menno T. P. van Laarhoven, Vincent J. W. Hellebrekers, Henrik G. van Hoef, Antonius J. M. Geurts, Wendy van Straaten, Wilco Nijenhuis, Florian Berger, Carlas S. Smith, Ihor Smal, and Lukas C. Kapitein. Outcome-driven microscopy: closed-loop optogenetic control of cell biology. <em>bioRxiv</em>, December 2024. URL: <a class="reference external" href="http://dx.doi.org/10.1101/2024.12.12.628240">http://dx.doi.org/10.1101/2024.12.12.628240</a>, <a class="reference external" href="https://doi.org/10.1101/2024.12.12.628240">doi:10.1101/2024.12.12.628240</a>.</p>
</div>
<div class="citation" id="id30" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>PCW+18<span class="fn-bracket">]</span></span>
<p>Antonio Z Politi, Yin Cai, Nike Walther, M Julius Hossain, Birgit Koch, Malte Wachsmuth, and Jan Ellenberg. Quantitative mapping of fluorescently tagged cellular proteins using fcs-calibrated four-dimensional imaging. <em>Nature Protocols</em>, 13(6):1445–1464, May 2018. URL: <a class="reference external" href="http://dx.doi.org/10.1038/nprot.2018.040">http://dx.doi.org/10.1038/nprot.2018.040</a>, <a class="reference external" href="https://doi.org/10.1038/nprot.2018.040">doi:10.1038/nprot.2018.040</a>.</p>
</div>
<div class="citation" id="id34" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>RHB+14<span class="fn-bracket">]</span></span>
<p>Sergi Regot, Jacob J. Hughey, Bryce T. Bajar, Silvia Carrasco, and Markus W. Covert. High-sensitivity measurements of multiple kinase activities in live single cells. <em>Cell</em>, 157(7):1724–1734, June 2014. URL: <a class="reference external" href="http://dx.doi.org/10.1016/j.cell.2014.04.039">http://dx.doi.org/10.1016/j.cell.2014.04.039</a>, <a class="reference external" href="https://doi.org/10.1016/j.cell.2014.04.039">doi:10.1016/j.cell.2014.04.039</a>.</p>
</div>
<div class="citation" id="id31" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>SZH+22<span class="fn-bracket">]</span></span>
<p>Sanjana Singh, Joanna Zukowska, Aliaksandr Halavatyi, Jonathan J. M. Landry, and Rainer Pepperkok. Phenotype-based single-cell transcriptomics reveal compensatory pathways involved in golgi organization and associated transport. <em>bioRxiv</em>, December 2022. URL: <a class="reference external" href="http://dx.doi.org/10.1101/2022.12.02.518815">http://dx.doi.org/10.1101/2022.12.02.518815</a>, <a class="reference external" href="https://doi.org/10.1101/2022.12.02.518815">doi:10.1101/2022.12.02.518815</a>.</p>
</div>
<div class="citation" id="id11" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>SLMP+20<span class="fn-bracket">]</span></span>
<p>Florian Sizaire, Gilles Le Marchand, Jacques Pécréaux, Otmane Bouchareb, and Marc Tramier. Automated screening of aurka activity based on a genetically encoded fret biosensor using fluorescence lifetime imaging microscopy. <em>Methods and Applications in Fluorescence</em>, 8(2):024006, February 2020. URL: <a class="reference external" href="http://dx.doi.org/10.1088/2050-6120/ab73f5">http://dx.doi.org/10.1088/2050-6120/ab73f5</a>, <a class="reference external" href="https://doi.org/10.1088/2050-6120/ab73f5">doi:10.1088/2050-6120/ab73f5</a>.</p>
</div>
<div class="citation" id="id16" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>SLB+25<span class="fn-bracket">]</span></span>
<p>Nicholas Sofroniew, Talley Lambert, Grzegorz Bokota, Juan Nunez-Iglesias, Peter Sobolewski, Andrew Sweet, Lorenzo Gaifas, Kira Evans, Alister Burt, Draga Doncila Pop, Kevin Yamauchi, Melissa Weber Mendonça, Genevieve Buckley, Wouter-Michiel Vierdag, Loic Royer, Ahmet Can Solak, Kyle I. S. Harrington, Jannis Ahlers, Daniel Althviz Moré, Oren Amsalem, Ashley Anderson, Andrew Annex, Peter Boone, Jordão Bragantini, Matthias Bussonnier, Clément Caporal, Jan Eglinger, Andreas Eisenbarth, Jeremy Freeman, Christoph Gohlke, Kabilar Gunalan, Hagai Har-Gil, Mark Harfouche, Volker Hilsenstein, Katherine Hutchings, Jessy Lauer, Gregor Lichtner, Ziyang Liu, Lucy Liu, Alan Lowe, Luca Marconato, Sean Martin, Abigail McGovern, Lukasz Migas, Nadalyn Miller, Hector Muñoz, Jan-Hendrik Müller, Christopher Nauroth-Kreß, David Palecek, Constantin Pape, Eric Perlman, Kim Pevey, Gonzalo Peña-Castellanos, Andrea Pierré, David Pinto, Jaime Rodríguez-Guerra, David Ross, Craig T. Russell, James Ryan, Gabriel Selzer, MB Smith, Paul Smith, Konstantin Sofiiuk, Johannes Soltwedel, David Stansby, Jules Vanaret, Pam Wadhwa, Martin Weigert, Jonas Windhager, Philip Winston, and Rubin Zhao. Napari: a multi-dimensional image viewer for python. 2025. URL: <a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.3555620">https://zenodo.org/doi/10.5281/zenodo.3555620</a>, <a class="reference external" href="https://doi.org/10.5281/ZENODO.3555620">doi:10.5281/ZENODO.3555620</a>.</p>
</div>
<div class="citation" id="id38" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>TWL13<span class="fn-bracket">]</span></span>
<p>Jared E. Toettcher, Orion D. Weiner, and Wendell A. Lim. Using optogenetics to interrogate the dynamic control of signal transmission by the ras/erk module. <em>Cell</em>, 155(6):1422–1434, December 2013. URL: <a class="reference external" href="http://dx.doi.org/10.1016/j.cell.2013.11.004">http://dx.doi.org/10.1016/j.cell.2013.11.004</a>, <a class="reference external" href="https://doi.org/10.1016/j.cell.2013.11.004">doi:10.1016/j.cell.2013.11.004</a>.</p>
</div>
<div class="citation" id="id4" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>TDBEA+25<span class="fn-bracket">]</span></span>
<p>Rémy Torro, Beatriz Díaz-Bello, Dalia El Arawi, Ksenija Dervanova, Lorna Ammer, Florian Dupuy, Patrick Chames, Kheya Sengupta, and Laurent Limozin. Celldetective: an ai-enhanced image analysis tool for unraveling dynamic cell interactions. <em>eLife</em>, March 2025. URL: <a class="reference external" href="http://dx.doi.org/10.7554/eLife.105302.1">http://dx.doi.org/10.7554/eLife.105302.1</a>, <a class="reference external" href="https://doi.org/10.7554/elife.105302.1">doi:10.7554/elife.105302.1</a>.</p>
</div>
<div class="citation" id="id24" role="doc-biblioentry">
<span class="label"><span class="fn-bracket">[</span>ZLX+23<span class="fn-bracket">]</span></span>
<p>Yian Zhao, Wenyu Lv, Shangliang Xu, Jinman Wei, Guanzhong Wang, Qingqing Dang, Yi Liu, and Jie Chen. Detrs beat yolos on real-time object detection. <em>bioRxiv</em>, 2023. URL: <a class="reference external" href="https://arxiv.org/abs/2304.08069">https://arxiv.org/abs/2304.08069</a>, <a class="reference external" href="https://doi.org/10.48550/ARXIV.2304.08069">doi:10.48550/ARXIV.2304.08069</a>.</p>
</div>
</div>
</div>
</section>
<section id="learn-more">
<h2>Learn more<a class="headerlink" href="#learn-more" title="Link to this heading">#</a></h2>
<p>This is just a simple starter to get you started.
You can learn a lot more at <a class="reference external" href="https://jupyterbook.org">jupyterbook.org</a>.</p>
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./."
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
</div>
                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#what-is-myst">What is MyST?</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sample-roles-and-directives">Sample Roles and Directives</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#citations">Citations</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#learn-more">Learn more</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By Smart Microscopy Working Group
</p>

  </div>
  
  <div class="footer-item">
    

  <p class="copyright">
    
      © Copyright 2023.
      <br/>
    
  </p>

  </div>
  
  <div class="footer-item">
    
  </div>
  
  <div class="footer-item">
    
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b"></script>
<script src="_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>